### PR TITLE
Changelog for v2.151.0

### PR DESCRIPTION
**Pull request by vEpiphyte**

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -6,6 +6,53 @@
...
+  (`#3364 <https://github.com/vertexproject/synapse/pull/3364>`_)
+- Update the minimum required version of the ``fastjsonschema`` library.
+  (`#3358 <https://github.com/vertexproject/synapse/pull/3358>`_)
+- Update tests and remove the use of deprecated functions for improved
+  Python 3.12 compatibility.
+  (`#3355 <https://github.com/vertexproject/synapse/pull/3355>`_)
+
+Bugfixes
+--------
+- Fixed a byg when parenting a View to another View where the bottom view has
```
Comment (MichaelSquires) [inline CHANGELOG.rst:42]:
```suggestion
- Fixed a bug when parenting a View to another View where the bottom view has
```

File: CHANGELOG.rst [hunk 2]
```diff
@@ -6,6 +6,53 @@
 Synapse Changelog
 *****************
 
+v2.151.0 - 2023-10-06
+=====================
+
+Features and Enhancements
+-------------------------
+
+- Add the Axon HTTP APIs to the Cortex. These API endpoitns use the Axon that
```
Comment (MichaelSquires) [inline CHANGELOG.rst:15]:
```suggestion
- Add the Axon HTTP APIs to the Cortex. These API endpoints use the Axon that
```

Comment (mikemoritz) [inline CHANGELOG.rst:15]: typo "endpoitns"

File: CHANGELOG.rst [hunk 3]
```diff
@@ -6,6 +6,53 @@
...
+  the Cortex is configured to use.
+  (`#3550 <https://github.com/vertexproject/synapse/pull/3550>`_)
+- Allow user defined functions in Storm to execute in ``readonly`` Storm
+  runtime.
+  (`#3552 <https://github.com/vertexproject/synapse/pull/3552>`_)
+- Clarify the Nexus ``IsReadOnly`` exception to include the common cause for
+  the error, which insufficent space on disk.
```
Comment (mikemoritz) [inline CHANGELOG.rst:22]: "which is"

File: CHANGELOG.rst [hunk 4]
```diff
@@ -6,6 +6,53 @@
...
+v2.151.0 - 2023-10-06
+=====================
+
+Features and Enhancements
+-------------------------
+
+- Add the Axon HTTP APIs to the Cortex. These API endpoitns use the Axon that
+  the Cortex is configured to use.
+  (`#3550 <https://github.com/vertexproject/synapse/pull/3550>`_)
+- Allow user defined functions in Storm to execute in ``readonly`` Storm
```
Comment (mikemoritz) [inline CHANGELOG.rst:18]: "in a"

File: CHANGELOG.rst (hunk 1 above)
Comment (mikemoritz) [inline CHANGELOG.rst:42]: typo "byg"

**Review by MichaelSquires — CHANGES_REQUESTED** [review]
found a couple of small typos

**Review by Cisphyx — APPROVED** [review]

**Pull request merged**